core/src/net/sf/openrocket/util/TestRockets.java

```diff
@@ -3,7 +3,6 @@
 import java.util.Random;
 
 import net.sf.openrocket.appearance.Appearance;
-import net.sf.openrocket.file.openrocket.OpenRocketSaver;
```

Comment:
This is needed for the dumpRocket() function in PR 603

Reply:
This change was fixed during rebases.  It's included, now.